src/deferred.js

```diff
@@ -48,6 +48,11 @@
 
           promise: function(obj) {
             return obj != null ? $.extend( obj, promise ) : promise
+          },
+      	  destory: function(){
```

Comment:
1. The indentation is slightly off.
2. This says “destory” instead of “destroy”.
3. Where is this function coming from? Is it supported by jQuery?